repos/system_upgrade/el7toel8/models/udev.py

```diff
@@ -0,0 +1,9 @@
+from leapp.models import Model, fields
+from leapp.topics import SystemInfoTopic
+
+
+class UdevAdmInfoData(Model):
+    topic = SystemInfoTopic
+
+    # Database export obtained by executing "udevadm info -e".
+    db = fields.String()
```

Comment:
Use please docstring to document particular fields instead of comment in this case.